--- a/bootloader/main.c
+++ b/bootloader/main.c
@@ -460,12 +460,6 @@ parse_failed:

 		if (ofw_reboot)
 		{
-			gfx_con.mute = false;
-			gfx_clear_grey(0x1B);
-			gfx_con_setpos(0, 0);
-			gfx_printf("\nRebooting to 100%% Stock OFW...\n");
-			msleep(1000);
-
 			power_set_state(REBOOT_BYPASS_FUSES);
 		}
 		else
@@ -632,12 +626,6 @@ parse_failed:

 		if (ofw_reboot)
 		{
-			gfx_con.mute = false;
-			gfx_clear_grey(0x1B);
-			gfx_con_setpos(0, 0);
-			gfx_printf("\nRebooting to 100%% Stock OFW...\n");
-			msleep(1000);
-
 			power_set_state(REBOOT_BYPASS_FUSES);
 		}
 		else
@@ -1072,12 +1060,6 @@ skip_list:

 		if (ofw_reboot)
 		{
-			gfx_con.mute = false;
-			gfx_clear_grey(0x1B);
-			gfx_con_setpos(0, 0);
-			gfx_printf("\nRebooting to 100%% Stock OFW...\n");
-			msleep(2000);
-
 			power_set_state(REBOOT_BYPASS_FUSES);
 		}
 		else
